en/docs/learn/configuring-data-sharing-for-secondary-users.md

```diff
@@ -0,0 +1,97 @@
+
+## Integrating with the bank back-end
+
+### Secondary Account Update API
+
+The Secondary Account Update API is defined below.
+
+The bank needs to invoke this endpoint to update the open banking solution regarding status changes happening on secondary user accounts. This endpoint persists the received secondary user-related information in the `OB_ACCOUNT_METADATA` table. When persisting the `accountId` of the secondary account and `secondaryUserId` parameters are used as a combined key and metadata will be stored as key-value pairs.
+
+```
+PUT - api/openbanking/account-type-mgt/account-type-management/secondary-accounts
+```
+
+This endpoint is secured with basic authentication.
+
+A sample request is given below:
+
+```
+curl --location --request PUT 'https://<IS_HOME>:9446/api/openbanking/account-type-mgt/account-type-management/secondary-accounts' \
+--header 'Authorization: Basic YWRtaW5Ad3NvMi5jb206d3NvMjEyMw==' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+   "data": [
+      {
+         "accountId":"30080098763500",
+         "secondaryUserId":"admin@wso2.com",
+         "otherAccountsAvailability":true,
+         "secondaryAccountInstructionStatus":"active"
+      },
+      {
+         "accountId":"30080098763501",
+         "secondaryUserId":"admin@wso2.com",
+         "otherAccountsAvailability":true,
+         "secondaryAccountInstructionStatus":"inactive"
+      }
+   ]
+}'
+```
+
+# Consent search API Modifications
+
+A new search query parameter named `secondaryAccountInfo` is introduced to the consent search API. The value of the `secondaryAccountInfo` parameter will be set to `true` to search for secondary account bounded consents related to a particular account holder. This search API will be used to obtain data needed for the secondary account consent page in the consent management dashboard. Then account holders will be able to view and manage such consents through the dashboard. 
+
+A sample request is given below:
+
+```
+GET https://<WSO2_OB_IAM_HOME>:9446/api/openbanking/consent/admin/search?secondaryAccountInfo=true?userIDs=<ACCOUNT_OWNER_USER_ID>
+```
+
+A sample response is given below:
+
+```json
+{
+   "clientId": "dIwYnqkohTHwOI1bUG0VpWxVNCoa",
+   "currentStatus": "authorized",
+   "createdTimestamp": 1683569588,
+   "recurringIndicator": true,
+   "authorizationResources": [...],
+   "updatedTimestamp": 1683569588,
+   "validityPeriod": 1683629555,
+   "consentAttributes": {
+      "ExpirationDateTime": "1683629555",
+      "commonAuthId": "1a25530b-cf2a-4341-8b27-5fd0a2d5557c",
+      "sharing_duration_value": "60000"
+      "consentId": "85d14688-abe5-4cc0-98c3-7f6af1d0a5e2",
+      "consentMappingResources": [....],
+      "secondaryAccountInfo": [{
+         "secondaryAccountUser": ”user1 @gold.com @carbon.super”,
```

Comment:
should corrected to ”user1@gold.com @carbon.super”

Reply:
Fixed via https://github.com/wso2/docs-ob-toolkit-cds/pull/78/commits/cae48427b88d4c488acaa5414de54c8e21565690.